helpers/transformers/web.js

```diff
@@ -0,0 +1,77 @@
+const { tint } = require('./utils');
+const cfonts = require('cfonts');
+const fs = require('fs');
+
+function build(BRAND) {
+	const cwd = `../../brands/${BRAND}`;
+
+	const { COLORS } = require(`${cwd}/tokens/colors`);
+	const { SPACING } = require(`${cwd}/tokens/spacing`);
+	const { LAYOUT } = require(`${cwd}/tokens/layout`);
+	const { TYPE } = require(`${cwd}/tokens/type`);
+
+	// COLORS
+	const tints = {};
+	Object.keys(COLORS).map(color => (tints[color] = { ...tint(COLORS[color]) }));
+
+	// TYPE
+	const brandfonts = {};
+	TYPE.brandfonts.map(font => {
+		if (font.name) {
+			if (!brandfonts['@font-face']) brandfonts['@font-face'] = [];
+
+			brandfonts['@font-face'].push({
+				fontFamily: font.name,
+				src: `url("${font.files.woff2}") format("woff2"), url("${font.files.woff}") format("woff")`,
+				weight: font.weight,
+				style: font.style,
+			});
```

Comment:
This won't do the thing you expect if this is being passed to emotion, it should be `something.push({ '@font-face': {fontFamily: font.name, ...theOtherStuff}})`

Reply:
I can't find the docs for how you add multiple `@font-face` declarations in object notation in emotion @mitchellhamilton ... I assumed it was the below. If it is not please let me know what it is.

```json
"@font-face": [
    {
      "fontFamily": "name",
      "src": "url(\"../font/dragonbold-webfont.woff2\") format(\"woff2\"), url(\"../font/dragonbold-webfont.woff\") format(\"woff\")",
      "weight": 400,
      "style": "normal"
    },
    {
      "fontFamily": "name",
      "src": "url(\"../font/dragonbold-bold-webfont.woff2\") format(\"woff2\"), url(\"../font/dragonbold-bold-webfont.woff\") format(\"woff\")",
      "weight": 700,
      "style": "normal"
    }
  ]
}
```

Reply:
It would have to be

```json
[{
  "@font-face":    {
      "fontFamily": "name",
      "src": "url(\"../font/dragonbold-webfont.woff2\") format(\"woff2\"), url(\"../font/dragonbold-webfont.woff\") format(\"woff\")",
      "weight": 400,
      "style": "normal"
    },
}, {
  "@font-face":{
      "fontFamily": "name",
      "src": "url(\"../font/dragonbold-bold-webfont.woff2\") format(\"woff2\"), url(\"../font/dragonbold-bold-webfont.woff\") format(\"woff\")",
      "weight": 700,
      "style": "normal"
    }
}]
```

Reply:
And how do you add that to the object of other style?

Reply:
This would work
```json
{ "": [
  {
    "@font-face": {
      "fontFamily": "name",
      "src": "url(\"../font/dragonbold-webfont.woff2\") format(\"woff2\"), url(\"../font/dragonbold-webfont.woff\") format(\"woff\")",
      "weight": 400,
      "style": "normal"
    }
  },
  {
    "@font-face": {
      "fontFamily": "name",
      "src": "url(\"../font/dragonbold-bold-webfont.woff2\") format(\"woff2\"), url(\"../font/dragonbold-bold-webfont.woff\") format(\"woff\")",
      "weight": 700,
      "style": "normal"
    }
  }
]}```

Or you could make the other style inside an array like this `[otherStyle, fonts]`